pkg/targetgroup/targetgroup.go

```diff
@@ -0,0 +1,101 @@
+package targetgroup
+
+import (
+	"github.com/common-fate/common-fate/pkg/storage/keys"
+	"github.com/common-fate/common-fate/pkg/types"
+	"github.com/common-fate/ddb"
+	"github.com/common-fate/provider-registry-sdk-go/pkg/providerregistrysdk"
+)
+
+type TargetGroup struct {
+	//user defined e.g. 'okta'
+	ID           string            `json:"id" dynamodbav:"id"`
+	TargetSchema GroupTargetSchema `json:"grouptargetSchema" dynamodbav:"groupTargetSchema"`
+	// reference to the SVG icon for the target group
+	Icon              string                   `json:"icon" dynamodbav:"icon"`
+	TargetDeployments []DeploymentRegistration `json:"targetDeployments" dynamodbav:"targetDeployments"`
+}
+
+type GroupTargetSchema struct {
```

Comment:
Can this just be called TargetSchema?

Reply:
This was decided as I found it painful referencing both `targetgroup.TargetSchema` and `providerregistrysdk.TargetSchema` in the same files.